### DISCUSSION TITLE
No response from GitHub Support

### DISCUSSION THREAD

**Discussion by ghost** [community/community#133701] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

After 3-4months I've received no response to my GitHub Support Ticket. My acc was suspended, i can't login, view my repos, nothing.

I have github education on that account, i did'nt break any rules idk why my acc was suspended?.

Github is Short staffed or is the ticketing system bugged?

Ticket ID: #2710390
Acc name: RevenzMind

**Comment by masriyey** [comment, marked as answer]
Hello,

Here are some steps you might consider to address the issue:

1. **Follow Up on Your Support Ticket**: If you haven’t already, try following up on your existing support ticket by replying to the original ticket confirmation email. Mention your ticket ID: #2710390.

2. **Check Email Filters**: Ensure that GitHub's emails are not going to your spam/junk folder, which might be causing you to miss their responses.

Here’s a template you could use for your follow-up message:

---

Dear GitHub Support Team,

I am writing to follow up on my support ticket ID: #2710390. My account (username: RevenzMind) was suspended several months ago, and I have not received any response regarding the reason for the suspension or steps to resolve it. I am unable to access my repositories, and this has significantly impacted my work.

I would appreciate it if you could expedite the review of my case or provide me with an update on the status of my support ticket. If any additional information is required from my side, please let me know, and I will promptly provide it.

Thank you for your attention to this matter. I look forward to your prompt response.

Best regards,

RevenzMind

---

I hope this helps you get the support you need. Good luck!

**Reply by SAHELISAHAA** [reply]
support team email?

**Reply by billythekid** [reply]
Hi @SAHELISAHAA 👋 

Support can only be contacted via https://support.github.com. 

There's no email address outside of opening a ticket.

Thanks,
Billy 🤠

**Reply by SAHELISAHAA** [reply]
@billythekid thanks but I already raised a ticket but my work is super urgent it is affecting my job a lot . I got several important projects on my GitHub need them urgently within 4 days orelse i will lose my job . Please tell me what to do . I am helpless.
Ticket ID: 2913646
username = @MAVRICK-1

**Reply by billythekid** [reply]
If you have a ticket open then Support will be in touch as soon as they can. There's no way to _skip the queue_, for fairness to everyone.

I was just replying to clarify the accepted answer above and your question about an email address. You've already done the right thing.

Best of luck for a speedy resolution.
